src/overlays/actors/ovl_Boss_Ganondrof/z_boss_ganondrof.c

```diff
@@ -61,71 +148,1395 @@ static ColliderCylinderInit D_80914CFC = {
     },
     { 20, 30, -20, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Boss_Ganondrof/func_80910640.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Boss_Ganondrof/func_80910680.s")
+// clang-format off
+static u8 sDecayMaskHigh[16 * 16] = {
+    1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,
```

Comment:
dont agree to turning of clang for this. at the very least, its not something we do in other places and need to discuss it on a larger scale

Reply:
while I understand your point, it just looks so nice like this.